chat-sdk/src/main/java/com/amazon/connect/chat/sdk/utils/CommonUtils.kt

```diff
@@ -70,6 +72,14 @@ class CommonUtils {
             val extension = fileName.substringAfterLast('.', "")
             return MimeTypeMap.getSingleton().getMimeTypeFromExtension(extension) ?: "*/*"
         }
+
+        fun createConnectParticipantConfiguration(): ClientConfiguration {
+            val clientConfiguration = ClientConfiguration()
+            val originalUserAgent: String = clientConfiguration.getUserAgent()
+            val userAgentSuffix = "AmazonConnect-Mobile Chat-Android SDK/" + BuildConfig.SDK_VERSION
```

Comment:
Question for long term - do we want to differentiate core SDK vs UI SDK users when we launch UI SDK?

Reply:
+1

Reply:
The proposed pattern is that in the future, we will create a hook in the SDK to append additional text to the user agent string.  In the scenario where we have a UI package, we would pass in an additional user agent string suffix.  The end UA string may look something like this:

```
aws-sdk-android/2.22.6 ... AmazonConnect-Mobile Chat-Android SDK/1.0.7 Chat-Android UI-SDK/1.0.0
```